docs/source/PyMOLplugin.rst

```diff
@@ -0,0 +1,233 @@
+    selection or trajectory and topology files by using the **Export CWS
+    Input Data** button in the **Water Selection** section.
+
+    **Number of snapshots**: Number of snapshots in the trajectory used
+    to generate the CWS input data file.
+
+  .. tab:: **Restart**
+
+    Restart a previous water clustering analysis. Provide the partial
+    results file and CWS restart data file for restarting the analysis.
+
+    **Partial Results File**: Browse and select the partial results file
+    for restarting the analysis. This file should contain the partial
+    results of the clustering analysis. For more information see
+    `ConservedWaterSearch website 
+    <https://conservedwatersearch.readthedocs.io/en/latest/examples.html>`_.
+    
+    **CWS restart data file**: Browse and select the CWS restart data
+    file. This file has the same structure as CWS input data file.
+    This data file should contain the clustering data set. If only
+    oxygens are clustered every row should contain the x, y, and z
+    coordinates of oxygen atoms for every frame to be studied. For
+    analysis of water types (recommended) each row should contain the
+    x, y, and z coordinates of oxygen and hydrogen atoms. For more
+    information see `ConservedWaterSearch documentation
+    <https://conservedwatersearch.readthedocs.io/en/latest/examples.html>`_.
+
+
+Water Selection
+---------------
+This section provides guidance on selecting water molecules for
+analysis. Proper water selection ensures that the analysis focuses on
+relevant waters in your structure. Users should take great care when
+choosing the waters of interest. Making an appropriate choice is
+crucial, as selecting a large number of water molecules can
+significantly slow down the analysis process. Typically, the most
+relevant water molecules are those situated near the binding site,
+proteins, or other surfaces. While you can analyze bulk waters, they
+tend to diffuse freely. If results are produced from these waters,
+especially those classified as WCW type, exercise caution when
+interpreting the data. 
+
+- **Water Selection Center**: Choose the method for selecting the center
+  of water molecules selection. Options include:
+
+.. tabs::
+
+  .. tab:: **Geometric Mean**
+
+    Provide a MDAnalysis selection string. The geometric mean of the
+    selected atoms will be used as the center. This selection is handled
+    by MDAnalysis whose selection language is similar in most instances
+    but not identical to PyMOL. For more information see `MDanalysis
+    <https://www.mdanalysis.org/docs/documentation_pages/selections.html#simple-selections>`_. 
+
+  .. tab:: **XYZ**
+
+    Specify the x, y, and z coordinates for center of water selection.
+
+
+.. tabs::
+  .. tab:: **Key Residue and Atom Names**
+
+    Users will have to provide residue name assigned to water molecules
+    and waters oxygen and hydrogen atom names. Alternatively, the plugin
```

Comment:
Maybe "Users will need to provide the residue name assigned to water molecules, as well as the atom names for oxygen and hydrogen within the water molecules."?